### PR TITLE
Move to ZonedDateTime. Close #375

### LINKED ISSUE #375: Check time datatypes and LocalDateTime (de)serialization

### PR DESCRIPTION
**Pull request by schleichardt**

### CODE REVIEW

**Comment by schleichardt** [comment]
@lauraluiz please review

File: sphere-common/src/main/java/io/sphere/sdk/json/ZonedDateTimeSerializer.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+final class ZonedDateTimeSerializer extends StdScalarSerializer<ZonedDateTime> {
+    static final long serialVersionUID = 0L;
+
+    public ZonedDateTimeSerializer() {
+        super(ZonedDateTime.class);
+    }
+
+    @Override
+    public void serialize(ZonedDateTime value, JsonGenerator gen, SerializerProvider arg2) throws IOException {
+        final String res = DateTimeFormatter.ISO_INSTANT.format(value.withZoneSameInstant(ZoneId.of("Z")));
```
Comment (lauraluiz) [inline sphere-common/src/main/java/io/sphere/sdk/json/ZonedDateTimeSerializer.java]: `ZoneOffset.UTC` or similar cannot be used, in order to make it more readable?

Comment (schleichardt) [inline sphere-common/src/main/java/io/sphere/sdk/json/ZonedDateTimeSerializer.java]: Cool, thanks. How did you find this?

Comment (lauraluiz) [inline sphere-common/src/main/java/io/sphere/sdk/json/ZonedDateTimeSerializer.java]: I use it in the type serializer for the search endpoint :P
https://github.com/sphereio/sphere-jvm-sdk/blob/master/sphere-common/src/main/java/io/sphere/sdk/search/TypeSerializer.java#L77

**Pull request merged**